src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7445,6 +7445,12 @@ components:
           type: string
         updatedDate:
           type: string
+        jobId:
+          type: string
+          description: The id for the job to create the snapshot once approved
+        createdSnapshotId:
```

Comment:
It's a shame that there's no way to add a description for a `$ref` (yet)

Reply:
Yeah, I'd love to throw some descriptions on here - eventually maybe openapi codegen will get there. (Or I guess we could PR to their repo, but I think that would be overly yak shaving)